torch/_dynamo/config.py

```diff
@@ -43,6 +43,21 @@
 # turn on/off DCE pass (deprecated: always true)
 dead_code_elimination = True
 
+# Enable or disable side effect replay after graph execution.
+# When False, mutations to Python objects (lists, dicts, attributes) won't be
+# replayed after the compiled graph runs. This can cause correctness issues
+# if your code depends on these mutations being visible. This should probably
+# never be False by default. At the moment, only export will need it.
+# [@compile_ignored: runtime_behaviour]
```

Comment:
What is this annotation for? First time I have seen it

Reply:
i actually don't know what it is. Just saw it was used everywhere else.